packages/amplify-e2e-tests/src/__tests__/import_s3_3.test.ts

```diff
@@ -38,9 +36,6 @@ describe('headless s3 import', () => {
     const shortId = getShortId();
     bucketNameToImport = `${bucketPrefix}${shortId}`;
 
-    const credentials = new aws.SharedIniFileCredentials({ profile: profileName });
-    aws.config.credentials = credentials;
```

Comment:
this was the only test with this still left in, and is not needed since the aws creds for the test acc are already available. Perhaps this was left in from local testing or before the configure_tests was added